lib/compress.js

```diff
@@ -4321,6 +4332,20 @@ merge(Compressor.prototype, {
                 })
             }).optimize(compressor);
         }
+        if (compressor.option("properties") && self.expression instanceof AST_Object) {
```

Comment:
You know what - `"unsafe"` alone should be enough. All the other object `"unsafe"` literal simplifications should be under an option called `"object_literals"` or `"objects"`.